eth2deposit/key_handling/key_derivation/path.py

```diff
@@ -20,7 +20,7 @@ def path_to_nodes(path: str) -> List[int]:
 
 def mnemonic_and_path_to_key(*, mnemonic: str, path: str, password: str) -> int:
     """
-    Returns the SK at position `path` secures with `password` derived from `mnemonic`.
+    Returns the SK at position `path`, secured with `password` derived from `mnemonic`.
```

Comment:
I guess you mean:
```suggestion
    Return the SK at position `path` that secured with `password` and derived from `mnemonic`.
```


Also, note that we [do not use the given `password` in production](https://github.com/ethereum/eth2.0-deposit-cli/blob/3ebc6ae1f06e5d8053d5cf55f91e7fd17fdb0a63/eth2deposit/credentials.py#L35-L36). `password` is only be set in tests. Perhaps add a note here?

Reply:
Indeed, the password is not exposed to a CLI user to prevent confusion with other passwords used. I have updated the documentation to reflect this.